src/engine/map/MapFunctions.cxx

```diff
@@ -15,9 +15,9 @@
 
 MapFunctions::MapFunctions()
 {
-  SignalMediator::instance().registerCbNewGame(Signal::slot(this, &MapFunctions::newMap));
+  // SignalMediator::instance().registerCbNewGame(Signal::slot(this, &MapFunctions::newMap));
   SignalMediator::instance().registerCbSaveGame(Signal::slot(this, &MapFunctions::saveMapToFile));
```

Comment:
why new and load game moved but save is not ?

Reply:
because save can be performed in mapfunctions, new game and load game also need to reset the gameplay managers to clean (and rebuild) existing grids